src/BaseRepository.php

```diff
@@ -197,4 +199,12 @@ protected function selectModels(array $condition, array $params = [])
 
 		return $models;
 	}
+
+	/**
+	 * @param BaseCollection $collection
+	 */
+	public function saveCollection(BaseCollection $collection)
+	{
+		$collection->map([$this, 'update']);
```

Comment:
Imagine having 1.000 items, we would call 1.000 times "DBA::update()", wouldn't it be better to have a single update query for it, would it? I imagine using the repository with "Worker", where there are a lot more used models than frontpage calls, or is this unrealistic?

Reply:
Bulk changes to a single column are better done with custom queries. Collections are better suited in pagination scenarios with 30-50 items max. Note that with the model original data optimization, not all model update would necessarily end up in an actual query.

Reply:
ok